test/integration/targets/prepare_vmware_tests/tasks/init_real_lab.yml

```diff
@@ -21,3 +21,7 @@
       when: setup_dvswitch is defined
     - include_tasks: setup_resource_pool.yml
       when: setup_resource_pool is defined
+    - include_tasks: setup_category.yml
```

Comment:
`setup_category.yml` is rather short, can we merge it with `setup_tag.yml`?

Reply:
In future, this might contains other category related stuff. So, I would keep this as is.

Reply:
:+1: